src/js/msp.js

```diff
@@ -204,25 +204,24 @@ var MSP = {
         this.message_buffer_uint8_view = new Uint8Array(this.message_buffer);
     },
     _dispatch_message: function(expectedChecksum) {
+        // Reset variables
+        this.message_length_received = 0;
```

Comment:
What is the reason this has been moved around?

Reply:
Had an issue before, placed it back now, keeps working. So fixed.